node/api/host.go

```diff
@@ -24,6 +24,12 @@ var (
 )
 
 type (
+	// ContractInfoGET contains the information that is returned after a GET request
+	// to /host/contracts - information for the host about stored obligations.
+	ContractInfoGET struct {
+		Contracts []modules.StorageObligation `json:"obligations"`
```

Comment:
generally these names should match -- either the Go field should be named `Obligations`, or the JSON field should be named `contracts`.

Reply:
Thanks, I will update the JSON to `contracts`